app/src/main/kotlin/com/passwordstore/android/ui/activity/SplashActivity.kt

```diff
@@ -4,18 +4,35 @@
  */
 package com.passwordstore.android.ui.activity
 
+import android.content.Intent
+import android.content.SharedPreferences
 import android.os.Bundle
 import android.view.ViewGroup
+import androidx.preference.PreferenceManager
 import com.passwordstore.android.databinding.ActivitySplashBinding
 import com.passwordstore.android.ui.EdgeToEdge
+import com.passwordstore.android.utils.Constants.IS_FIRST_RUN
 
 class SplashActivity : BaseActivity() {
+
     private lateinit var binding: ActivitySplashBinding
+    private lateinit var prefs: SharedPreferences
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        prefs = PreferenceManager.getDefaultSharedPreferences(this)
         binding = ActivitySplashBinding.inflate(layoutInflater)
         EdgeToEdge.setUpRoot(binding.root as ViewGroup)
         setContentView(binding.root)
+        if (prefs.getBoolean(IS_FIRST_RUN, true)) {
+            startFirstRunFlow()
+        } else {
```

Comment:
unneeded

Reply:
Which part ?

Reply:
The empty `else` clause, unless something's supposed to go there that's gonna come later.

Reply:
We'll have code for syncing repos / other stuff we need to do to launch the MainActivity there.